GPU/GPUTracking/DataTypes/CalibdEdxContainer.h

```diff
@@ -32,6 +32,22 @@
 namespace o2::tpc
 {
 
+/// flags to set which corrections will be loaded from the CCDB
+enum class Calibsdedx : unsigned short {
+  calTopologySpline = 1 << 0,  ///< flag for a topology correction using splines
+  calTopologyPol = 1 << 1,     ///< flag for a topology correction using polynomials
+  calThresholdMap = 1 << 2,    ///< flag for using threshold map
+  calGainMap = 1 << 3,         ///< flag for using the gain map to get the correct cluster charge
+  calResidualGainMap = 1 << 4, ///< flag for applying residual gain map
+  calResidualGain = 1 << 5,    ///< flag for residual dE/dx gain correction
```

Comment:
Naming convention is capital case: https://rawgit.com/AliceO2Group/CodingGuidelines/master/naming_formatting.html?showone=Enumerator_Names#Enumerator_Names
I'm wondering if we should change `calResidualGain` to `CalTimeGain`, to be consisten with the CCDB path.